ticdc/ticdc-faq.md

```diff
@@ -262,3 +264,23 @@ Currently, TiCDC is not yet fully compatible with TiDB Lightning and BR. Therefo
 ## After a changefeed resumes from pause, its replication latency gets higher and higher and returns to normal only after a few minutes. Why?
 
 When a changefeed is resumed, TiCDC needs to scan the historical versions of data in TiKV to catch up with the incremental data logs generated during the pause. The replication process proceeds only after the scan is completed. The scan process might take several to tens of minutes.
+
+## How should I deploy TiCDC to replicate data between two TiDB cluster located in different regions?
+
+It is recommended that you deploy TiCDC in the downstream TiDB cluster. If the network latency between the upstream and downstream is large, for example, more than 100 ms, the latency produced when TiCDC executes SQL statements to the downstream might increase dramatically due to issues of the MySQL transmission protocol. This results in a decrease in system throughput. However, deploying TiCDC in the downstream can greatly ease this problem.
+
+## What is the order of executing DML and DDL statements?
+
+The execution order is: DML -> DDL -> DML. To ensure that the table structure is correct when DML events are executed downstream during data replication, it is necessary to coordinate the execution order of DDL and DML statements. Currently, TiCDC adopts a simple approach: It replicates all DML statements before the DDL ts to downstream first, and then replicates DDL statements.
+
+## How should I check whether the upstream and downstream data are consistent?
+
+If the downstream is a TiDB or MySQL cluster, it is recommended that you compare the data by using [sync diff inspector](/sync-diff-inspector/sync-diff-inspector-overview.md).
+
+## Replication of a single table can only be run on a single TiCDC node. Will it be possible to use multiple TiCDC nodes to replicate data of multiple tables?
+
+This feature is under development. In the future, TiCDC can replicate data change logs by TiKV Region, which means scalable processing capability.
```

Comment:
It may be not proper to state future feature plan.
```suggestion
## Replication of a single table can only be run on a single TiCDC node. Will it be possible to use multiple TiCDC nodes to replicate data of multiple tables?

This feature is currently not supported, which might be supported in a future release. By then, TiCDC might replicate data change logs by TiKV Region, which means scalable processing capability.
```